src/com/carolinarollergirls/scoreboard/view/ScoreBoard.java

```diff
@@ -60,18 +59,15 @@
     public List<Team> getTeams();
     public Team getTeam(String id);
 
-    public Ruleset _getRuleset();
-    public String getRuleset();
-
     public Settings getSettings();
-    // Frontend (i.e. javascript only) settings.
-    public FrontendSettings getFrontendSettings();
+
+    public Rulesets getRulesets();
 
     public Stats getStats();
 
     public XmlScoreBoard getXmlScoreBoard();
 
-    public static final String FRONTEND_SETTING_CLOCK_AFTER_TIMEOUT = "ScoreBoard.ClockAfterTimeout";
+    public static final String SETTING_CLOCK_AFTER_TIMEOUT = "ScoreBoard.ClockAfterTimeout";
 
     public static final String SETTING_NUMBER_PERIODS = "Rule." + Clock.ID_PERIOD + ".Number";
```

Comment:
> Since the values are accessed through the Ruleset Object now and in order to seperate them from the previous FRONTEND_SETTINGs, the constants' prefix should be changed to RULE_.

Done.

> Also as they are no longer mixed with other settings, the prefix "Rule." in the String has become superfluous.

I was leaving that to a future cleanup.